install/rpm/templates/web/nginx/php-fpm/NodeJS.stpl

```diff
@@ -0,0 +1,76 @@
+server {
+    listen      %ip%:%proxy_port%;
+    server_name %domain_idn% %alias_idn%;
+    return      301 https://%domain_idn%$request_uri;
+}
+
+server {
+    listen      %ip%:%proxy_ssl_port%;
```

Comment:
Should be web_proxy_ssl

Reply:
https://github.com/darkworks/hestiacp/blob/darkworks-nodejs/install/rpm/templates/web/nginx/php-fpm/NodeJS.stpl#L2